packages/client/src/provider/in-memory-provider/in-memory-provider.ts

```diff
@@ -59,16 +59,11 @@ export class InMemoryProvider implements Provider {
       .filter(([key, value]) => this._flagConfiguration[key] !== value)
       .map(([key]) => key);
 
-    this.status = ProviderStatus.STALE;
```

Comment:
No need for stale here - this is a configuration change (a change sourced from the provider itself) not a change from setting context, etc.

This corresponds to a change coming directly from a management system (no need for STALE/READY here).

Reply:
Please double-check the server implementation to make sure it doesn't have the same issue.

Reply:
I did.